app/src/main/java/com/github/fribourgsdp/radio/data/view/UserProfileActivity.kt

```diff
@@ -122,9 +122,11 @@ open class UserProfileActivity : MyAppCompatActivity(), KeepOrDismissPlaylistDia
         if(user.isGoogleUser){
             db.setUser(user.id,user)
         }else{
+            val userPlaylists = user.getPlaylists()
             val userWithoutPlaylist = user
-            userWithoutPlaylist.removePlaylists(user.getPlaylists())
+            userWithoutPlaylist.removePlaylists(userPlaylists)
             db.setUser(user.id,userWithoutPlaylist)
+            user.addPlaylists(userPlaylists)
```

Comment:
Why does it save the name in the database even if you are not connected to Google?

Reply:
The idea was to save the name of the user on the server. After discussion with @VictorSchneuwly, it seems that we don't use it at all, as a user gives its name directly to the lobby. This behavior and uniformisation of what a offline user can do will be treated in my other task, where I have to add the offline/online behavior.